src/components/TransactionPreview/HistoryPreview.tsx

```diff
@@ -66,32 +66,25 @@ const HistoryPreview = ({ crossChainAction }: TransactionPreviewInterface) => {
     // Convert gas wei to ETH
     const gasInETHAmount = gasCost / 1e18;
 
-    // Convert to USD
-    let gasFees = gasInETHAmount * crossChainAction.estimated.usdPrice;
-    let gasFeesNative = gasInETHAmount;
+    // calculate gas fees
+    const gasfees = gasInETHAmount * asset.gasUsed;
```

Comment:
please follow camelCase naming standards

```suggestion
    const gasFees = gasInETHAmount * asset.gasUsed;
```